index.bs

```diff
@@ -12291,6 +12291,33 @@ to Consider</a>
 	recording or music making environment). Excessive latency decreases usability and may be
 	an accessibility issue.
 
+    Fingerprining via the sample rate of the {{AudioContext}} is also possible.  We
```

Comment:
If we're doing it for the rate, we should do it for `outputLatency` and `maxChannelCount`.

Reply:
`maxChannelCount` should clearly be fixed to 2.  But what about `outputLatency`?  Not sure what value to use for that.  10 ms?

Reply:
Gecko does this https://searchfox.org/mozilla-central/source/dom/media/webaudio/AudioContext.cpp#557-566, those are common numbers gathered from looking at a bunch of machines, and only make sense for Gecko, I assume: it's not buffering at all and it's up to date, because we know what OS API/buffer sizes we use underneath, and what a good "fake" latency figure should be.

Reply:
I guess, based on your link, that we just want to say that the `outputLatency` needs to be handled but in a browser-specific way, allowing browsers to choose good default values for each supported OS as needed.

Reply:
I agree, that's the best solution.

Reply:
Fingerprinting of `outputLatency` is covered in the previous paragraph: 
> Fingerprinting via latency is also possible; it might be possible to
> deduce this from {{baseLatency}} and {{outputLatency}}. Mitigation
> strategies include adding jitter (dithering) and quantization so
> that the exact skew is incorrectly reported.

So I don't think we need to add anything else about this in this PR.

Just need to add a sentence about maxChannelCount.

Reply:
maxChannelCount set to 2.